rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Bothe `:book` and `:person` should have an `attr_accessor`. It will be nice to fix this.